RecoCTPPS/PixelLocal/interface/RPixDetPatternFinder.h

```diff
@@ -0,0 +1,62 @@
+/*
+ *
+* This is a part of CTPPS offline software.
+* Author:
+*   Fabrizio Ferro (ferro@ge.infn.it)
+*   Enrico Robutti (robutti@ge.infn.it)
+*   Fabio Ravera   (fabio.ravera@cern.ch)
+*
+*/
+#ifndef RecoCTPPS_PixelLocal_RPixDetPatternFinder_H
+#define RecoCTPPS_PixelLocal_RPixDetPatternFinder_H
+
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+
+#include "DataFormats/Common/interface/DetSetVector.h"
+#include "DataFormats/Common/interface/DetSet.h"
+
+#include "DataFormats/CTPPSReco/interface/CTPPSPixelRecHit.h"
+#include "DataFormats/CTPPSDetId/interface/CTPPSPixelDetId.h"
+
+
+#include "Geometry/VeryForwardGeometryBuilder/interface/CTPPSGeometry.h"
+#include "CLHEP/Vector/ThreeVector.h"
+#include "CLHEP/Vector/RotationInterfaces.h"
```

Comment:
If I'm not mistaken, these are driven by the CTPPS geometry interfaces.
@ianna 
Was there a good reason to chose CLHEP instead of GeomDet-like interface like in tracking geometries?